contracts/libraries/wrapppers/MultiRequest.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.0;
+
+import "../../SnappBase.sol";
+
+
+contract MultiRequest is SnappBase {
```

Comment:
nit: This is not really a library, is it? Should we just create a test folder inside `contracts` and place it there? Also, I'd slightly prefer the name `MultiFlux`, but no strong feeling. 

If you wanted to be super cool, you could even make this thing target contract agnostic by exposing a single method that takes the target contract, the encoded calldata (method id and and inputs as byte array) and calls `address.call(calldata)` `multiplier` times.

This might be useful in other projects too.

Reply:
Yes, I would love to make these suggestions happen, but am not currently aware how. Show me?

Reply:
https://github.com/gnosis/mock-contract#mocking-methods--arguments shows you how you can use `encodeAbi` in js to get the encoded byte array of a method call.

Inside your multi contract you would do something like: 

```
function callMultiple(address contract, bytes calldata) {
  for (i=0, i<multiplier, i++) {
    contract.call(calldata)
  }
}
```

@rmeissner might also be able to help you with specific questions while I'm out.

Reply:
Awesome. I have started a small repo that is now working on functions that don't have any parameters. Will try and get back to this again soon and move it in to gnosis once it's in better shape with more features.